update/Pos/Update/Core/Types.hs

```diff
@@ -141,7 +144,7 @@ newtype SystemTag = SystemTag { getSystemTag :: Text }
   deriving (Eq, Ord, Show, Generic, Buildable, Hashable, Lift, Typeable)
 
 systemTagMaxLength :: Integral i => i
-systemTagMaxLength = 10
+systemTagMaxLength = 15
```

Comment:
No, revert it please. This change is basically a hardfork.
If you apply @georgeee's suggestion to retain backwards compatibility, this change won't be necessary (existing tags are less than 10 characters long).

Reply:
Thanks for pointing that out, this would have been a costly distraction.